training.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: training
+title: Training Materials
+---
+
+# Foreman training materials
+
+This page lists the materials available for those wishing to study/practice working
+with Foreman, as an alternative style to reading the manual. 
+
+## Netways Foreman Training Course
```

Comment:
I'd remove 'Foreman' from the title, it gets kind of repetitive (2 titles with it separated by 2 lines) :smile: